### PR TITLE
Docs: OptionButton

### PR DESCRIPTION
**Pull request by jameswestman**
Completed the documentation and made some existing descriptions a bit clearer.

### CODE REVIEW

File: doc/classes/OptionButton.xml [hunk 1]
```diff
@@ -109,12 +110,14 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the ID of the selected item, or 0 if no item is selected.
```
Comment (Calinou) [inline doc/classes/OptionButton.xml]:
```suggestion
				Returns the ID of the selected item, or [code]0[/code] if no item is selected.
```

File: doc/classes/OptionButton.xml [hunk 2]
```diff
@@ -192,14 +201,15 @@
...
-				Sets the text of an item at index [code]idx[/code].
+				Sets the text of the item at index [code]idx[/code].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="action_mode" type="int" setter="set_action_mode" getter="get_action_mode" override="true" enum="BaseButton.ActionMode" default="0" />
 		<member name="align" type="int" setter="set_text_align" getter="get_text_align" override="true" enum="Button.TextAlign" default="0" />
 		<member name="selected" type="int" setter="_select_int" getter="get_selected" default="-1">
+			The index of the currently selected item, or -1 if no item is selected.
```
Comment (Calinou) [inline doc/classes/OptionButton.xml]:
```suggestion
			The index of the currently selected item, or [code]-1[/code] if no item is selected.
```

File: doc/classes/OptionButton.xml [hunk 3]
```diff
@@ -182,6 +190,7 @@
 			<argument index="1" name="metadata" type="Variant">
 			</argument>
 			<description>
+				Sets the metadata of an item. Metadata may be any type and can be used to store extra information about an item, such as an external string ID.
```
Comment (Calinou) [inline doc/classes/OptionButton.xml]: This looks more grammatically correct to me:

```suggestion
				Sets the metadata of an item. Metadata may be of any type and can be used to store extra information about an item, such as an external string ID.
```

File: doc/classes/OptionButton.xml [hunk 4]
```diff
@@ -139,7 +144,8 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
-				Select an item by index and make it the current item.
+				Selects an item by index and makes it the current item.
+				This will work even if the item is disabled.
```
Comment (Calinou) [inline doc/classes/OptionButton.xml]: This could be put on the same paragraph as the line above (i.e. don't put a line break).

**Review by Calinou — CHANGES_REQUESTED** [review]
Content looks good to me. I made some suggestions about formatting :slightly_smiling_face:

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**